src/test/scala/com/github/xiaodongw/swagger/finatra/SampleApp.scala

```diff
@@ -29,7 +29,7 @@ object SampleApp extends HttpServer {
   override def configureHttp(router: HttpRouter) {
     router
       .filter[CommonFilters]
-      .add[WebjarsController]
+      .add(new WebjarsController)
```

Comment:
Have you verified if the sample app still work? I tried it and it can not show the swagger UI.